src/status.rs

```diff
@@ -177,24 +177,24 @@ impl FromIter for Status {
                     match result.nextsong {
                         None => {
                             result.nextsong = Some(QueuePlace {
-                                pos: try!(line.1.parse()),
-                                id: Id(0),
-                                prio: 0,
-                            })
+                                                       pos: try!(line.1.parse()),
+                                                       id: Id(0),
+                                                       prio: 0,
+                                                   })
                         }
                         Some(ref mut place) => place.pos = try!(line.1.parse()),
                     }
                 }
                 "time" => {
+                    let mut splits = line.1.splitn(2, ':').map(|v| v.parse().map_err(ParseError::BadInteger).map(Duration::seconds));
```

Comment:
I pulled this line out of the block inside the `try!` because the resulting line was longer than `rustfmt` would allow.  It was failing with the message:

```
Rustfmt failed at /src/rust/rust-mpd/src/status.rs:190: line exceeded maximum length (maximum: 140, found: 156) (sorry)
```